pkg/kubelet/dockershim/doc.go

```diff
@@ -15,4 +15,4 @@ limitations under the License.
 */
 
 // Docker integration using pkg/kubelet/apis/cri/runtime/v1alpha2/api.pb.go
-package dockershim
+package dockershim // import "k8s.io/kubernetes/pkg/kubelet/dockershim"
```

Comment:
I'm guessing it's to stop importing via `github.com/kubernetes/kubernetes`, but I'm not clear when that would happen.  @mysunshine92 ?

Reply:
@dixudx @justinsb I have made this update  because other packages did the same

// Handlers for pod lifecycle events and interfaces to integrate
// with kubelet admission, synchronization, and eviction of pods.
package lifecycle // import "k8s.io/kubernetes/pkg/kubelet/lifecycle"